src/components/Map.js

```diff
@@ -31,20 +31,16 @@ const Map = ({
   if (!isObjEmpty(mapsObj)) {
     const { map } = mapsObj;
     map.addListener("center_changed", function () {
```

Comment:
![Screen Cast 2020-06-13 at 7 24 59 PM](https://user-images.githubusercontent.com/4306/84583286-cc7d5480-adab-11ea-80f8-95c6d63e9f31.gif)

The updating of the map with new pins is choppy in Safari (I wasn't able to pan in Chrome because or the "two fingers" error) as seen in the attached GIF. I wonder if that's because we're using `center_changed` instead of `idle` as the event we're watching? We discussed using `idle` on the last call because it means that the map is for sure done panning. As you can see in the GIF below, `center_changed` can fire multiple times as the user is dragging.

![Screen Cast 2020-06-13 at 7 30 24 PM](https://user-images.githubusercontent.com/4306/84583344-76f57780-adac-11ea-9d6b-e647730ce776.gif)

I would have _thought_ that the work we did to make sure requests don't stack up (i.e. that a new request is only fired once the previous one has completed) would have mitigated this, but I think it's worth trying to see if using the `idle` event makes this all work more smoothly.

Reply:
Just a preliminary check with some similar logic from the pendingPromise PR, and it does not alleviate the choppy loading of pins.

Reply:
OK, I'll leave it up to you 2 whether you want to dig into this now or create an issue for us to improve this later. Thanks for trying!

Reply:
I created [this issue](https://github.com/the-collab-lab/tcl-pilot-2020-05/issues/29) for the choppy pins. Resolving this convo so we’re clear to merge!